### PR TITLE
Added more improvements about this plone theme package

### PR DESCRIPTION
**Pull request by macagua**
@davilima6 can you check these improvements and if all are Ok, can merged it?

### CODE REVIEW

File: plonetheme/motion/profiles/uninstall/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<metadata>
```
Comment (davilima6) [inline plonetheme/motion/profiles/uninstall/metadata.xml]: GS doesn't read/need metadata.xml so I suggest removing this one

**Review by davilima6 — APPROVED** [review]
Thanks! Very welcome PR.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+*.wpr
+.*.cfg
+.installed.cfg
+.mr.developer.cfg
+.eggs/
+.hg/
+.bzr/
+.svn/
+.project
+.pydevproject
```
Comment (davilima6) [inline .gitignore]: I suggest not including project specific ignores like IDEs or OSs as well as VCS not in use

File: docs/HISTORY.txt [hunk 3]
```diff
@@ -1,10 +1,14 @@
 Changelog
 =========
 
-trunk (unreleased)
-------------------
+0.7 (unreleased)
```
Comment (davilima6) [inline docs/HISTORY.txt:4]: I think we can update the date here and add a release in GitHub

File: plonetheme/motion/profiles/uninstall/metadata.xml (hunk 1 above)
Comment (macagua) [reply plonetheme/motion/profiles/uninstall/metadata.xml]: @davilima6 done with commit 6aa52ac

File: .gitignore (hunk 2 above)
Comment (macagua) [reply .gitignore]: @davilima6 done with commit abbe3cf

File: docs/HISTORY.txt (hunk 3 above)
Comment (davilima6) [reply docs/HISTORY.txt:4]: done in https://github.com/davilima6/plonetheme.motion/releases/tag/0.7

thanks!!

**Comment by macagua** [comment]
@davilima6 really thanks for this merge!

**Comment by macagua** [comment]
Hi @davilima6 

I like to make a new release at PyPI, then You can make a release? and is possible add my PyPI user as Collaborator? for help you to manage this package at PyPI , If in this moment, you can't not to create a new release, I will can do it for you!

**Pull request merged**